public/v2/apps/etherpad.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/ether/etherpad-lite/blob/develop/doc/docker.md ",
+    "displayName": "Etherpad",
+    "description": "Etherpad is a real-time collaborative editor for the web",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+            "image": "etherpad/etherpad:$$cap_etherpad_version",
+            "ports": [
+                "9001:9001"

```

Comment:
Why is there a port mapping as well as containerHttpPort?

Reply:
a brain fart, sorry